README.md

```diff
@@ -1,4 +1,4 @@
-![Serial Keel Build](https://github.com/torsteingrindvik/serial-keel/.github/workflows/basics.yml/badge.svg)
+![Serial Keel Build](https://github.com/torsteingrindvik/serial-keel/actions/workflows/basics.yml/badge.svg)
```

Comment:
How does this work?
It looks like it would just display an svg image, but there is no such image in the repo.
Does github produce it via the CI system?